pending/change-session-id.pod

```diff
@@ -0,0 +1,107 @@
+=head1 change_session_id
+
+It is considered good security practice to change session ID on any change
+of privilege level, for example on login.  It is also a requirement of a
+number of established security standards so is something you should be
+doing in your web applications.
+
+=head2 Session destruction
+
+Destroying a session in L<Dancer2> has always been possible using
+L<Dancer2::Core::App/destroy_session> thus:
```

Comment:
"thusly" is a colloquial alternative to "thus" in UK English so "thus" is preferred in formal language. Perhaps "like so" would be better.